### ISSUE TITLE
React canary: Mega Menu animation (Only on iPhone) Masthead (Default)

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#5787]
## Detailed description

Issue:
After expanding the hamburger menu, the following menu options are visually available. Products & Solutions, Services & Consulting, Learn & Support, Explore more
But Screen reader identifies the entire menu options which are not visually available. 
Note: working fine in React staging

Expected:
The menu options which are visually not available should be hidden from the screen reader

**Comment by annawen1** [comment]
Hey @Praveen-sr I believe this specific issue has already been fixed, I had opened a ticket earlier to get it in the April release: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/5771

Can you confirm? Thanks!

**Comment by Praveen-sr** [comment]
@annawen1 Re tested this on React canary, issue has been fixed. Working as expected. 
cc @RobertaJHahn
